src/net_processing.cpp

```diff
@@ -1286,6 +1325,11 @@ bool PeerManagerImpl::GetNodeStateStats(NodeId nodeid, CNodeStateStats& stats) c
         ping_wait = GetTime<std::chrono::microseconds>() - peer->m_ping_start.load();
     }
 
+    if (peer->m_tx_relay != nullptr) {
+        stats.fRelayTxes = WITH_LOCK(peer->m_tx_relay->cs_filter, return peer->m_tx_relay->fRelayTxes);
+        stats.minFeeFilter = peer->m_tx_relay->minFeeFilter.load();
+    }
```

Comment:
eeb0e84fc5c259dfd00f7673ad4a51582b100282: any reason to make this code more fragile by removing the `else` in the move? The `else` used to set those to `false`, and `0`, respectively.

Reply:
I don't think I agree that this is more fragile. The default is that there connection isn't being used to relay transactions, and the feefilter is set to zero. Those stats are only updated if the node is relaying transactions/has set a feefilter.

I've reverted this to use if/else and remove the default initialization, since that's closer to the current code.